op-program/host/kvstore/disk.go

```diff
@@ -1,93 +1,62 @@
 package kvstore
 
 import (
-	"encoding/hex"
-	"errors"
 	"fmt"
-	"io"
-	"os"
-	"path"
+	"runtime"
 	"sync"
 
+	"github.com/cockroachdb/pebble"
 	"github.com/ethereum/go-ethereum/common"
 )
 
-// read/write mode for user/group/other, not executable.
-const diskPermission = 0666
-
-// DiskKV is a disk-backed key-value store, every key-value pair is a hex-encoded .txt file, with the value as content.
+// DiskKV is a disk-backed key-value store, with PebbleDB as the underlying DBMS.
 // DiskKV is safe for concurrent use with a single DiskKV instance.
-// DiskKV is safe for concurrent use between different DiskKV instances of the same disk directory as long as the
-// file system supports atomic renames.
 type DiskKV struct {
```

Comment:
We'll need to add a `Close()` function to this now and wire it into the overall lifecycle to ensure it gets closed when op-program shuts down.